metro.config.js

```diff
@@ -1,16 +1,16 @@
-const { getDefaultConfig } = require("metro-config");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { getDefaultConfig } = require("expo/metro-config");
 
-module.exports = (async () => {
-  const {
-    resolver: { sourceExts, assetExts },
-  } = await getDefaultConfig();
-  return {
-    transformer: {
-      babelTransformerPath: require.resolve("react-native-svg-transformer"),
-    },
-    resolver: {
-      assetExts: assetExts.filter((ext) => ext !== "svg"),
-      sourceExts: [...sourceExts, "svg"],
-    },
-  };
-})();
+const config = getDefaultConfig(__dirname);
+
+config.transformer = {
```

Comment:
changed this file to match what is suggested by [expo](https://docs.expo.dev/guides/customizing-metro/). the end result is the same, but eas build has a warning in our original version and no warning in this version